SpatialGDK/Source/SpatialGDK/Private/Interop/CrossServerRPCSender.cpp

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/CrossServerRPCSender.h"
+
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Utils/SpatialMetrics.h"
+
+namespace SpatialGDK
+{
+CrossServerRPCSender::CrossServerRPCSender(ViewCoordinator& InCoordinator, USpatialMetrics* InSpatialMetrics)
+	: Coordinator(&InCoordinator)
+	, SpatialMetrics(InSpatialMetrics)
+{
+}
+
+void CrossServerRPCSender::SendCommand(const FUnrealObjectRef& InTargetObjectRef, UObject* TargetObject, UFunction* Function,
```

Comment:
Minor nit - unless there's an overload set prefer by value for passing move only stuff.